README.rst

```diff
@@ -51,6 +51,40 @@ substring of all the keys in the document and returns any values which match.
 
 For example:
 
+.. code-block:: python
+
+from nested_lookup import nested_lookup
+
+my_document = {
+    'name' : 'Russell Ballestrini',
+    'email_address' : 'test1@example.com',
+    'other' : {
+        'secondary_email' : 'test2@example.com',
+        'EMAIL_RECOVERY' : 'test3@example.com',
+    },
+},
+
+results = nested_lookup(
+    key = 'mail',
+    document = my_document,
+    wild = True
+)
+
+print(results)
+['test1@example.com', 'test2@example.com', 'test3@example.com']
+
+
+output
```

Comment:
I would move the above and combine the examples in the same code block.

Reply:
IU'm not sure how to achieve that. Assuming I create a single code block with both usages, where should I put it? Should I create an Example section at the bottom of the document and put it there? Should I leave the code block where the original one is and use the `output` option before having introduced it?